routes/web.php

```diff
@@ -287,37 +283,29 @@ function (): void {
                 Route::get('faq', 'FaqController')->name('faq');
 
                 /* Static - Privacy Policy */
-                Route::get(
+                Route::view(
                     'privacy',
-                    function () {
-                        return view(
-                            'common/static_privacy',
-                            [
-                                'privacy' => Lang::get('common/privacy'),
-                                'custom_breadcrumbs' => [
-                                    'home' => route('home'),
-                                    Lang::get('common/privacy.title') => '',
-                                ],
-                            ]
-                        );
-                    }
+                    'common/static_privacy',
+                    [
+                        'privacy' => Lang::get('common/privacy'),
+                        'custom_breadcrumbs' => [
+                            'home' => LaravelLocalization::localizeUrl('/'),
```

Comment:
Composer/PHP throws an error here when you try and use the `route()` helper to reference a named route. I assume it's because it's not actually defined yet when it tries to cache the entire routes file.